pkg/controller/virtualmachineimage/importer_pod.go

```diff
@@ -41,6 +41,10 @@ const (
 	ImportPodImage = "kubevirt/cdi-importer:v1.13.0"
 	// ImportPodVerbose indicates log level of the import pod
 	ImportPodVerbose = "1"
+	// SourceVolumeName provides a const to use for creating volumes in pod specs
```

Comment:
need to be const? we will never change this value

Reply:
This value is used in many places, so I made it const.